site/src/components/PageHeader.tsx

```diff
@@ -64,7 +65,15 @@ const PageHeader: React.SFC<PageHeaderProps> = ({
             </div>
           )}
           <div className={styles.mainSection}>
-            <h1 className={styles.headingText}>{headingText}</h1>
+            <div className={styles.headingTextContainer}>
+              <Heading
+                variant={headingOnly ? "display-0" : "heading-1"}
```

Comment:
Zen Heading 1 was too small for the front page (headingOnly), but Display 0 was too big for non front page pages.